README.md

```diff
@@ -7,7 +7,26 @@
 
 Frontend Service Discovery on AWS is a solution to handle frontend releases on AWS using the [frontend discovery pattern](https://github.com/awslabs/frontend-discovery).
 
-The frontend discovery pattern improves the development experience when developing, testing, and delivering micro-frontends by making use of a shareable configuration describing the entry point of micro-frontends, as well as additional metadata that is used to deploy in every environment safely using Canary Releases.
+The **frontend discovery pattern** improves the development experience when developing, testing, and delivering micro-frontends by making use of a shareable configuration describing the entry point of micro-frontends, as well as additional metadata that is used to deploy in every environment safely using Canary Releases.
+
```

Comment:
shall we add something here regarding multi-environments and ephemeral environments and how this solution fits in them?

Reply:
Ok I added a small chapter about that...not sure if that's what you meant? Thoughts?